include/misc/__assert.h

```diff
@@ -81,7 +81,7 @@
 
 #if __ASSERT_ON
 #include <misc/printk.h>
-void assert_post_action(void);
+void assert_post_action(const char *file, unsigned int line);
 
 #define __ASSERT_LOC(test)                               \
 	printk("ASSERTION FAIL [%s] @ %s:%d\n",    \
```

Comment:
If we're going to pass file/line to assert_post_action, might as well move this printk there too?